mlrun/serving/system_steps.py

```diff
@@ -375,6 +379,10 @@ def __init__(self, output_stream=None, **kwargs):
         super().__init__(**kwargs)
         stream = self.context.stream if self.context else None
         self.output_stream = output_stream or getattr(stream, "output_stream", None)
+        if self.output_stream is None:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "output_stream is None meaning not provided and " "not part of contex"
```

Comment:
Something bad happened to this string. I think it would actually be better to just not use `getattr`, because we'll have a bit more information from the error message (like what type `stream` was).